_data/nvram.yaml

```diff
@@ -1,12 +1,29 @@
 ---
 - name: AAPL,ignore
+- platforms:
+  - macOS T2
+  name: ApChipID
+  scopes:
+  - 94b73556-2197-4702-82a8-3e1337dafbfb
+- type: :boolean
+  platforms:
+  - macOS T2
+  description: Controls trust in Microsoft's PCA 2011 cert for Apple Secure Boot
+  name: AppleSecureBootWindowsPolicy
+  scopes:
+  - 94b73556-2197-4702-82a8-3e1337dafbfb
 - name: BOSCatalogURL
 - name: BaseSystemAccessabiliityFeatures
 - name: BluetoothInfo
 - name: BluetoothUHEDevices
 - name: DClr_override
 - name: DebugBluetoothHIDShim
-- name: DefaultBackgroundColor
+- platforms:
```

Comment:
Will improve the sorter to ensure that the entries keys are in canonical order as well (ID, description, scopes e.g.)